### PR TITLE
Update d.style-guides.md

### PR DESCRIPTION
**Pull request by jgreen44**
Typos and grammatical errors fixed.

### CODE REVIEW

**Comment by jgreen44** [comment]
@pamgoodrich : Do you mind taking a look at the changes?

**Review by mnaumanali94 — APPROVED** [review]

**Comment by mnaumanali94** [comment]
Oops :O 
Sorry! @pamgoodrich

**Comment by pamgoodrich** [comment]
@mnaumanali94 : Are you apologizing for merging doc changes I didn't make or approve? :) That's fine in this case, but Sydney and I are working in this area, so it would be good to let me know when you make changes to these files. @jgreen44

**Comment by pamgoodrich** [comment]
But looking at the changes... thank you, @jgreen44. Some bad typos in there.

File: 5.-governance/d.style-guides.md [hunk 1]
```diff
@@ -27,18 +27,18 @@ You might have noticed the APIs across your ecosystem use infinite different err
...
 
 Sometimes these style guides are published publicly, which can give useful insight into what big companies and government organizations consider to be a "good API" for them. [API Stylebook.com > Design Guidelines](http://apistylebook.com/design/guidelines/) is a collection of these style guides if you want to take a look and get some ideas.
 
-The trouble with these text-based documents is that they are large, terse documents which developers rarely read. If developers _do_ read them cover to cover, the chanced of them remembering everything is pretty slip. Even if they somehow remember 100% of the words written down, that knowledge gets out-of-date when changes and additions to the style guide are made. 
+The trouble with these text-based documents is that they are large, terse documents that developers rarely read. If developers _do_ read them cover to cover, the chances of them remembering everything is pretty slip. Even if they somehow remember 100% of the words written down, that knowledge gets out-of-date when changes and additions to the style guide are made.
```
Comment (jgreen44) [inline 5.-governance/d.style-guides.md:41]: @pamgoodrich : Slip should be "slim"

**Comment by mnaumanali94** [comment]
Haha yes. I'll make sure to not merge stuff you haven't looked at moving forward 😬

**Comment by pamgoodrich** [comment]
We'll be rewriting this topic for SSG. Haven't gotten there yet, but we will soon. (@mnaumanali94 : we need to figure out the balance of leaving the existing stuff in alongside the new stuff for early access. Sydney started with a new topic for the overview, so we can have both features until we fully GA SSG.)

**Pull request merged**